patches/0004-Use-crypto-backends.patch

```diff
@@ -2452,6 +2457,19 @@ index 00000000000000..573896b9c1e6a8
 +// the underlying functions because the TLS 1.3 KDF does not have a standard of
 +// its own.
 +
++func checkFIPS140Only[Hash hash.Hash](h func() Hash, key []byte) error {
```

Comment:
this is function is copy-pasted from `./src/crypto/hkdf/hkdf.go`

any refactoring suggestions? export the function?

Reply:
`tls13.ExpandLabel` is an internal API that's not reachable by the user, but only by `crypto/tls`. That package only uses FIPS-compliant parameters when calling `tls13.ExpandLabel`, so there is nothing to check here. You can remove this function.

Reply:
thanks, force pushed an update: https://github.com/microsoft/go/pull/1654/commits/c76587805a1d197c3eb8a667229c868c0f4f9196